EIPS/eip-918.md

```diff
@@ -1,27 +1,21 @@
 ---
 eip: 918
 title: Mineable Token Standard
-author: Jay Logelin <jlogelin@alumni.harvard.edu>, Infernal_toast <admin@0xbitcoin.org>, Michael Seiler <mgs33@cornell.edu>, Brandon Grill <bg2655@columbia.edu>
+description: A specification for standardized Mineable Tokens that uses a Proof of Work algorithm for distribution.
+author: Jay Logelin (@jlogelin), Infernal_toast <admin@0xbitcoin.org>, Michael Seiler <mgs33@cornell.edu>, Brandon Grill <bg2655@columbia.edu>
+status: Last Call
```

Comment:
```suggestion
status: Draft
```
This was the last status of the EIP before being moved to stagnant, so it should return to draft before moving to last call.